MediaWikiTelegramBot.js

```diff
@@ -122,29 +155,46 @@ tgBot.onText( /\/untranslated/, function ( msg, match ) {
 
     debug( userID, 'in onText untranslated' );
 
+    // Get the messages...
     mwApi.getUntranslatedMessages( languageCode, messageCollection => {
-        debug( userID, 'in getUntranslatedMessages' );
+        // debug( userID, 'in getUntranslatedMessages' );
 
+        // Check if a user with that ID exists, creates it if not
         if ( userStatus[userID] === undefined ) {
             userStatus[userID] = {};
         }
+
+        // Set defaults
         userStatus[userID]['messages'] = messageCollection;
         userStatus[userID]['currentMwMessageIndex'] = 0;
 
-        debug( userID, 'received messageCollection ' + JSON.stringify(
-            userStatus[userID]['messages'],
-            null,
-            2
-        ) );
+        // debug( userID, 'received messageCollection ' + JSON.stringify(
```

Comment:
That's why we have a debug function: So we could show debugging information only when it's needed. The debug function should have different levels. Currently it only has 1 and 0. It should also have level 2, which would show "extra" information.